ItsME/Presentation/Common/Cells/TextViewCell.swift

```diff
@@ -52,13 +54,21 @@ class TextViewCell: UITableViewCell {
             make.edges.equalToSuperview().inset(10)
             make.height.equalTo(textViewHeight)
         }
+
+        textView.addSubview(placeholderLabel)
+        placeholderLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.left.right.equalToSuperview()
+            make.bottom.equalToSuperview()
```

Comment:
```suggestion
            make.directionalEdges.equalToSuperview()
```
leading, trailing 까지 고려해서 위의 코드로 변경하면 깔끔할 것 같습니다!